src/hooks/useForceUpdate.ts

```diff
@@ -7,5 +7,5 @@ import { useReducer } from "react";
 const reducerAlwaysReturningNewObj = () => ({});
 
 export default function useForceUpdate() {
-  return useReducer(reducerAlwaysReturningNewObj)[1];
+  return useReducer(reducerAlwaysReturningNewObj, reducerAlwaysReturningNewObj)[1];
```

Comment:
??

Reply:
@sahariko it is mandatory according to typescript